modules/dnsmasq_dhcp/metadata.yaml

```diff
@@ -21,55 +23,47 @@ setup:
       description: |
         The following options can be defined globally: update_every, autodetection_retry.
       folding:
-        title: All options
+        title: Config options
         enabled: true
       list:
         - name: update_every
           description: Data collection frequency.
           default: 1
-          required: no
+          required: false
         - name: autodetection_retry
           description: Re-check interval in seconds. Zero means not to schedule re-check.
           default: 0
-          required: no
+          required: false
         - name: leases_path
-          description: Path to dnsmasq dhcp leases file.
-          default: "-"
-          required: no
+          description: Path to dnsmasq DHCP leases file.
+          default: "/var/lib/misc/dnsmasq.leases"
+          required: false
         - name: conf_path
           description: Path to dnsmasq configuration file.
-          default: "-"
-          required: no
+          default: "/etc/dnsmasq.conf"
+          required: false
         - name: conf_dir
           description: Path to dnsmasq configuration directory.
-          default: "-"
-          required: no
+          default: "/etc/dnsmasq.d,.dpkg-dist,.dpkg-old,.dpkg-new"
+          required: false
     examples:
+      folding:
+        title: Config
+        enabled: true
       list:
         - name: Basic
           description: An example configuration.
-          folding:
-            title: Example
-            enabled: true
           data: |
             jobs:
               - name: dnsmasq_dhcp
                 leases_path: /var/lib/misc/dnsmasq.leases
                 conf_path: /etc/dnsmasq.conf
                 conf_dir: /etc/dnsmasq.d
-        - name: Multi-instance
```

Comment:
Mutli-instance for this collector has no sense, it can't be several instances of Dnsmasq on the same host